cfg/virtme-ng.conf

```diff
@@ -1,4 +1,11 @@
 {
-    "default_opts" : {
+    "default_opts": {},
+    "systemd": {
+        "masks": [
+            "/usr/lib/systemd/system-generators/systemd-fstab-generator",
+            "/usr/lib/systemd/system-generators/systemd-cryptsetup-generator",
+            "/usr/lib/systemd/system/auditd.service",
+            "/usr/lib/systemd/system/getty@.service"
```

Comment:
Can you please add comments for what the masks are used?

For example, it's not to obvious why `serial-getty@.service` is not masked.

Reply:
systemd-fstab-generator: it's responsible to mount all the devices from fstab, so if the user users args.root=/ then it will freeze the boot while waiting for the devices.
systemd-cryptsetup-generator: same thing, but this time it disables mounting also the encrypted drives he may have.
auditd: it was bringing a lot of failures if not used with --rw
getty@: the code in this PR effectively creates a customized serial-getty@.service file and force its usage by means of setting SYSTEMD_UNIT_PATH, so masking all the regular getty@ makes sense (so it removes any error that might flood the stream, e.g. if the user has autologin set for getty@tty1)

Reply:
Instead of masking the audit service it might be cleaner to just disable auditing in general (there is a kernel cmdline arg for that)

Reply:
Thanks, can you please add this as comments to the code? Otherwise no one will remember :)

Reply:
> Instead of masking the audit service it might be cleaner to just disable auditing in general (there is a kernel cmdline arg for that)

But then it wouldn't be configurable. I.e. the user might enable auditing by removing the mask and deal with the errors.

> Thanks, can you please add this as comments to the code? Otherwise no one will remember :)

Sure, but since its a JSON config. file, where do you suggest it goes? In the README?

Reply:
Oh no, JSON :D In the commit message at least?

Reply:
Added in https://github.com/arighi/virtme-ng/pull/280/commits/4ffb83c45a2782f631b7ec48bfa07edca902ec4c